cfn-resources/alert-configuration/test/README.md

```diff
@@ -0,0 +1,72 @@
+# Alert Configurations 
+
+## Impact 
+ - Alert Configuration L1 CDK constructor
+
+## Manual QA
+
+#### Resources needed to run the manual QA
+- Atlas Project
+
+All resources are created as part of `cfn-testing-helper.sh`
+
+### Steps
+
+#### Prerequisites
```

Comment:
For general steps, prerequisites and general success criteria it might be better to have another .md file and use links to redirect to that as in the manual QA doc. From a maintenance perspective it will be easier to make changes. rather than repeat them in each resource which is what we have seen usually happen in the past.

Reply:
Happy to move the common steps to a different REAMDE.md file.

Do you have any preferences regarding where this README.md file should live and be called?

1. Creating a new MANUALQA.md in the root of the repo
2. Adding the step in the main [README.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/README.md)
3. ?

Going for option 1 if I don't get any reply. Thanks!

Reply:
Updated

Reply:
We're flexible. I'd add a `TESTING.md` with a section for manual qa prereqs, but not a strong preference.